src/System.Net.Requests/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uap-Windows_NT;
```

Comment:
@davidsh let us know if that doesn't work correctly (with the netstandard configuration). I would expect this library to be packaged with the uap configuration of S.N.Primitives so everything should work.